### PR TITLE
Added some generic questions to FAQ

### PR DESCRIPTION
**Pull request by jorgensel**
To keep a FAQ while transitioning to new website setup, and avoid word and excel sheets

### CODE REVIEW

File: vipps-checkout-api-faq.md [hunk 1]
```diff
@@ -25,7 +25,45 @@ For more common questions, see:
...
+### How do I sign up for Checkout?
+To sign up for Checkout, you need to create an account on the Vipps website and follow the instructions to set up your account. You will need to provide information about your business and bank account details.
+
+### What kind of support is available for Checkout users?
+We provide support through our website, email, and through our extensive partner network. There is also a knowledge base available on the website that contains answers to frequently asked questions.
+
+## Payment Methods
+
+### What payment methods are available with Checkout?
+Vipps Checkout supports a range of payment methods, including Vipps, credit and debit cards. Local payment methods will also become availabel in select markets during 2024, e.g. bank payments in Finland.
```
Comment (rebekaburnett) [inline vipps-checkout-api-faq.md:42]:
```suggestion
Vipps Checkout supports a range of payment methods, including Vipps, credit and debit cards. Local payment methods will also become available in select markets during 2024, e.g. bank payments in Finland.
```

File: vipps-checkout-api-faq.md [hunk 2]
```diff
@@ -25,7 +25,45 @@ For more common questions, see:
...
+### What fees are associated with using Checkout?
+Vipps Checkout charges a percentage fee for the payment volume processed. Our standard prices can be found [here](https://vipps.no/alle-priser/bedrift/).
+
+## Integration and Functionality
+
+### Can I integrate Checkout with my existing website or online store?
+Yes, Vipps Checkout can be integrated with most major website and e-commerce platforms, including Shopify, WooCommerce, and Magento. Check out our [list of plugins](https://developer.vippsmobilepay.com/docs/plugins/) for more information.
+
+### How long does it take for payments to be paid out with Checkout?
+You will typically get your money on your account within three (3) banking days. So money paid to you on a Monday will be on your account on Wednesday, Payment on Friday will be on your account on Tuesday. [See more details here](https://developer.vippsmobilepay.com/docs/settlements/).
```
Comment (rebekaburnett) [inline vipps-checkout-api-faq.md:61]:
```suggestion
You will typically get your money on your account within three (3) banking days. So money paid to you on a Monday will be on your account on Wednesday, and payment on Friday will be on your account on Tuesday. [See more details here](https://developer.vippsmobilepay.com/docs/settlements/).
```

File: vipps-checkout-api-faq.md [hunk 3]
```diff
@@ -25,7 +25,45 @@ For more common questions, see:
...
+### What fees are associated with using Checkout?
+Vipps Checkout charges a percentage fee for the payment volume processed. Our standard prices can be found [here](https://vipps.no/alle-priser/bedrift/).
+
+## Integration and Functionality
+
+### Can I integrate Checkout with my existing website or online store?
+Yes, Vipps Checkout can be integrated with most major website and e-commerce platforms, including Shopify, WooCommerce, and Magento. Check out our [list of plugins](https://developer.vippsmobilepay.com/docs/plugins/) for more information.
+
+### How long does it take for payments to be paid out with Checkout?
+You will typically get your money on your account within three (3) banking days. So money paid to you on a Monday will be on your account on Wednesday, and payment on Friday will be on your account on Tuesday. [See more details here](https://developer.vippsmobilepay.com/docs/settlements/).
```
Comment (rebekaburnett) [inline vipps-checkout-api-faq.md:61]:
```suggestion
You will typically get your money on your account within three (3) banking days. So money paid to you on a Monday will be on your account on Wednesday, and payment on Friday will be on your account on Tuesday. See [Settlements](https://developer.vippsmobilepay.com/docs/settlements/) for more details.
```

File: vipps-checkout-api-faq.md [hunk 4]
```diff
@@ -25,7 +25,45 @@ For more common questions, see:
...
+
+### How does Checkout compare to other online payment solutions?
+Checkout offers Vipps as it is supposed to work, card payments, user information retrieval and shipping selection in one well designed bundle, and copares favorably, or at least we think that, to other online payment solutions.
```
Comment (rebekaburnett) [inline vipps-checkout-api-faq.md:64]:
```suggestion
Checkout offers Vipps as it is supposed to work with card payments, user information retrieval, and shipping selection all in one well-designed bundle, and compares favorably to other online payment solutions.
```

**Review by rebekaburnett — APPROVED** [review]

**Comment by rebekaburnett** [comment]
My review isn't enough since I pushed some comments.

**Comment by jorgensel** [comment]
I'm happy with your changes @rebekaburnett , but it seems like I can't review my own PR

**Review by SvenErikPy — APPROVED** [review]

**Pull request merged**